config/environments/development.rb

```diff
@@ -11,7 +11,14 @@
   config.cache_classes = !!ENV["PROFILE"]
 
   # :file_store is used by default when no cache store is specifically configured.
-  config.cache_store = :memory_store if !!ENV["PROFILE"]
+  if !!ENV["PROFILE"]
+    config.cache_store = :redis_cache_store, {
+      driver: :hiredis,
+      url: ENV.fetch("OFN_REDIS_URL", "redis://localhost:6379/0"),
+      namespace: "cache",
+      expires_in: 90.minutes
```

Comment:
why this? If we configure Redis as LRU there's no need for it, right?

Reply:
Actually I guess we don't _currently_ need to use Redis in dev, we could just stick to `:memory_store` for now?

Adding an expiration here in dev just makes it more explicitly ephemeral; Redis doesn't need to carefully keep hold of lots of cache entries around on my laptop (and use memory), right?

Reply:
ah, I missed this was dev config :+1: